### ISSUE TITLE
Update winget packages on release

### ISSUE THREAD

**Issue by hymccord** [Azure/azure-functions-core-tools#3667]
The `Microsoft.Azure.FunctionsCoreTools` package on winget can lag behind release until it's updated on the winget-packages repo (mostly manually by others). The chocolatey is sync'd with release.

I'd be happy to PR it myself.

It'd be the same as [microsoft/PowerToys/.github/workflows/package-submission.yml](https://github.com/microsoft/PowerToys/blob/main/.github/workflows/package-submissions.yml)

Or does it need to be another route that doesn't involve GitHub Actions?

**Comment by Frulfump** [comment]
Yeah still waiting on 4.4.1 but https://github.com/Azure/azure-functions-dotnet-worker/issues/3152#issuecomment-3528744669 doesn't seem to show winget as a tracked distribution channel which is a real bummer.
4.4.1 is stuck on this PR https://github.com/microsoft/winget-pkgs/pull/312504
